weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableQuery.ts

```diff
@@ -80,19 +82,50 @@ export const useCallsForQuery = (
     }
   }, [callResults.length, callsStats.loading, callsStats.result, offset]);
 
+  const costFilter: CallFilter = calls.loading
```

Comment:
memo this - each re-render is referentially uniq

Reply:
also, you can just do an empty list in first case (simpler code) since you skip later)